### ISSUE TITLE
Provide documentation in Finnish

### ISSUE THREAD

**Issue by ronja-ui** [teragrep/doc_01#86]
**Description**
To increase accessibility the official documentation should be provided in Finnish as well. 
 
**Use case or motivation behind the feature request**
There has been requests to have the official documentation of Teragrep in Finnish. The documentation could be provided through docs.teragrep.com or as a separate PDF depending on the schedule and resources. 
 
**Related issues**

 
**Additional context**
